onnxruntime/test/python/onnxruntime_test_ort_trainer.py

```diff
@@ -67,7 +67,7 @@ def create_ort_trainer(gradient_accumulation_steps,
                         allreduce_post_accumulation,
                         use_simple_model_desc=True,
                         loss_scaler=None,
-                        partition_optimizer=False):
+                        deepspeed_zero=0):
```

Comment:
I am wondering if we want to make this as explicit "deepspeed_zero_stage" ?
We might need other zero configs latter...

Reply:
Thanks, I'll update.  Longer term I expect we'll need to think how to handle a growing set of config settings more generally also.